### PR TITLE
Spike

### PR DESCRIPTION
**Pull request by lingtran**
Add initial routes, buttons and icons to ease into React and have initial design.
Login button takes user directly to home screen without Facebook login for now.

Would appreciate thoughts on architecture so far and any suggestions on how to break out components.

Next steps will be to fill in homescreen with stubbed data and break out the components into own files. And tests.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,13 +1,93 @@
-var React = require('react');
-var ReactDOM = require('react-dom');
+const React = require('react');
+const ReactDOM = require('react-dom');
```
Comment (jwashke) [inline lib/index.js:2]: Yay ES6

**Comment by jwashke** [comment]
My only suggestion is to add a readme explaining what the project is and how to run it locally.

File: lib/index.js [hunk 2]
```diff
@@ -1,13 +1,93 @@
...
+          <Link to="/home"><ActionButton id="login-button" text="Login" className="col-5" /></Link>
+          <ActionButton className="col-5" id="signup-button" text="Sign up" />
+        </div>
+      </div>
+    );
+  }
+};
+
+// buttons
+class ActionButton extends React.Component {
```
Comment (thomschlereth) [inline lib/index.js:42]: @lingtran Just a question, not a review. Is `extends React.Component {` the way you use react with es6? Is `Component` a React class or one that you created?

 I like how broken out your classes are, and that your naming is explicit. I don't know React but it looks good 👍 .

Comment (lingtran) [inline lib/index.js:42]: @thompickett Yes, I am use ES6 syntax! And thanks for the feedback regarding how the components are broken out. I am going to try restructuring with a parent Application component, per @julsfelic's recommendation. 😄

**Comment by lingtran** [comment]
@jwashke thanks for the feedback! will create issue cards for the comment.

**Pull request merged**